docs/integration-test-helpers/masonry/MasonryContainer.js

```diff
@@ -304,14 +366,16 @@ export default class MasonryContainer extends Component<Props, State> {
     }
 
     if (noScroll) {
-      dynamicGridProps.scrollContainer = () => undefined;
+      dynamicGridProps.scrollContainer = undefined;
```

Comment:
Why this change? This is a runtime change.

Reply:
@dangerismycat 

scrollContainer can't be `() => void`, it should be `() => HTMLElement`

But it can be optional / undefined.

Got a better approach for this?

<img width="450" alt="Screenshot by Dropbox Capture" src="https://user-images.githubusercontent.com/10593890/234576732-dbda7a13-d8b6-474d-bf6e-97335436b6ed.png">

Reply:
Ah ok, that makes sense. This (and similar changes) are fine, then. Thanks for the explanation 🙂